README.md

```diff
@@ -62,3 +62,73 @@ import vos
 client = vos.Client()
 client.listdir('vos:jkavelaars')
 ```
+
+## Development
+A virtual environment (**venv**) is recommended to set up external dependencies without installing them system-wide. Following [these instructions](http://docs.python-guide.org/en/latest/dev/virtualenvs/), install **virtualenv**:
+```
+$ pip install virtualenv
+```
+
+Next, create, and activate a local **venv** (this example uses **bash**):
+```
+$ virtualenv venv
+$ source venv/bin/activate
+
+```
+
+Finally, use **pip** to install missing external dependencies into this subdirectory:
+```
+$ pip install -r requirements.txt
```

Comment:
Having the == in the requirements.txt file will work for 'development' as we don't want to become dependent of new releases without specifically deciding to do so... but for the distribution we really should mean >= .  Of course eventually the future versions may not be incompatible with the older one.